data/successors/successor weapons.txt

```diff
@@ -0,0 +1,389 @@
+		"hit effect" "bimodal switch impact"
+		"inaccuracy" 10
+		"velocity" 30
+		"lifetime" 6
+		"random lifetime" 3
+		"shield damage" 48
+		"hull damage" 20
+
+effect "bimodal switch impact"
+	sprite "effect/bimodal switch impact"
+		"frame rate" 12
+		"no repeat"
+	lifetime 30
+	"velocity scale" 0.02
+
+outfit "Drag Cannon"
+	category "Guns"
+	series "Guns: Successors"
+	index 20
+	cost 455000
+	thumbnail "outfit/drag cannon"
+	licenses
+		"High Houses"
+	"mass" 12
+	"outfit space" -12
+	"weapon capacity" -12
+	"gun ports" -1
+	weapon
+		sound "drag cannon"
+		"fire effect" "moonbeam fleck" 2
+		"hit effect" "drag cannon impact" 3
+		"velocity" 0
+		"lifetime" 0
+		"range override" 375
+		"velocity override" 25
+		"burst count" 6
+		"burst reload" 6
+		"reload" 45
+		"firing energy" 30
+		"firing heat" 55
+		"cluster"
+		"submunition" "Drag Projectile" 3
+	description `The Successors have a historical affinity for projectile weaponry which persists to the modern day. The Drag Cannon fires a short-ranged, highly inaccurate blast of projectiles that not only deal substantial damage but also temporarily disrupt the target's engines.`
+	description `	While it is an inelegant design in comparison to the time-tested "Ojet" Coilgun, and consequently less appreciated, the Drag Cannon has found a place on the Successors' fast attack craft primarily as a tool for slowing down enemy vessels so that they can be engaged by a larger fleet.`
+
+outfit "Drag Projectile"
+	weapon
+		sprite "projectile/drag cannon"
+			"frame rate" 20
+			"no repeat"
+		"hit effect" "drag cannon impact"
+		"inaccuracy" 20
+		"velocity" 25
+		"lifetime" 10
+		"random lifetime" 5
+		"slowing damage" 0.45
+		"shield damage" 52
+		"hull damage" 48
+
+effect "drag cannon impact"
+	sprite "effect/drag cannon impact"
+		"frame rate" 10
+		"no repeat"
+	lifetime 20
+	"velocity scale" 0
+	"random velocity" 1
+
+outfit "Successor Pulse Laser"
+	category "Guns"
+	series "Guns: Successors"
+	index 30
+	cost 69000
+	thumbnail "outfit/successor pulse laser"
+	licenses
+		"Successor"
+	"mass" 10
+	"outfit space" -10
+	"weapon capacity" -10
+	"gun ports" -1
+	weapon
+		sprite "projectile/successor pulse laser"
+			"random start frame"
+			"frame rate" 5
+		sound "successor pulse laser"
+		"hit effect" "successor pulse laser impact"
+		"inaccuracy" 0.4
+		"velocity" 490
+		"lifetime" 1
+		"burst count" 9
+		"burst reload" 1
+		"reload" 6
+		"firing energy" 11
+		"firing heat" 15
+		"shield damage" 10.5
+		"hull damage" 12
+		"piercing" 0.05
+	description "This shield-penetrating laser is one of the few weapons manufactured independently in Successor space, rather than under the auspices and restrictions of the High Houses. As a consequence of this, it is neither especially powerful nor efficient, but its low cost and space requirements make it a nonetheless relatively appealing defensive choice for those without access to the High Houses' more potent weaponry."
+
+outfit "Pulse Laser Turret"
```

Comment:
Traverse seems very reasonable for a turret of this size. Other than that, see the comment above on the gun version.